### ISSUE TITLE
Error in AnyLogic 8.8.5

### ISSUE THREAD

**Issue by sajious** [t-wolfeadam/AnyLogic-Pypeline#72]
![Screenshot 2024-04-20 134915](https://github.com/t-wolfeadam/AnyLogic-Pypeline/assets/52618691/9b1ea26d-9529-495b-90f3-7f0d8d77c874)
I got this error, when I'm using AnyLogic 8.8.5 while I add this library, it will show me use AnyLogic 8.8.3 or higher.

**Comment by t-wolfeadam** [comment]
Despite what it seems, it's because the current Pypeline was built for AnyLogic 8.8.6 (the 8.8.3 is the version of the file format, not AnyLogic itself...something I really wish they'd change...). 

In short, you'll need to upgrade AnyLogic or recompile the library for your version (and 'downgrade' the examples if you want to open them). I'm actually surprised I can't find previous documentation for doing this, so I just added a section to the troubleshooting guide which explains more: [Unable to add the library](https://github.com/t-wolfeadam/AnyLogic-Pypeline/wiki/Troubleshooting#unable-to-add-the-library).

Closing as answered, but feel free to ask follow-ups.
